mne/io/fil/tests/test_fil.py

```diff
@@ -25,6 +25,21 @@
 )
 
 
+def _set_bads_tsv(chanfile, badchan):
+    """Update channels.tsv by setting target channel to bad."""
+    data = []
+    with open(chanfile) as f:
```

Comment:
can you please explicitly pass an encoding here and below?

Reply:
I did in the orignal code, but ruff seems to be auto formatting them out...?